pkg/minikube/out/out.go

```diff
@@ -66,10 +66,20 @@ type V map[string]interface{}
 
 // T writes a stylized and templated message to stdout
 func T(style StyleEnum, format string, a ...V) {
+	if style == Option {
+		Infof(format, a...)
+		return
+	}
 	outStyled := ApplyTemplateFormatting(style, useColor, format, a...)
 	String(outStyled)
 }
 
+// Infof is used for informational logs (options, env variables, etc)
+func Infof(format string, a ...V) {
```

Comment:
I find Info confusing, that when I am supposed to use Info vs T.
Either we should update out.T to be something else or how about naming it `out.TabbedInfo`
or `out.DisplayOption` or `out.SubInfo`

Reply:
Since the JSON struct is of type `Info` I think it may be more confusing if the function name and the struct name don't match.

Technically it's ok if someone uses T instead of Info, since T will just defer to Info in the correct case (when style=Option). This is more so just so that the code is cleaner for JSON output

Reply:
ok thats convincing enough.